src/main/kotlin/pt/up/fe/ni/website/backend/model/Account.kt

```diff
@@ -56,6 +58,24 @@ class Account(
     @OneToMany(cascade = [CascadeType.ALL], fetch = FetchType.EAGER)
     val websites: List<@Valid CustomWebsite>,
 
+    @JoinColumn
+    @ManyToMany(cascade = [CascadeType.ALL], fetch = FetchType.EAGER)
```

Comment:
Should this cascade? I think it means we'll delete the roles when an account is deleted